src/fmu/dataio/dataio.py

```diff
@@ -932,8 +932,8 @@ class AggregatedData:
     """Instantate AggregatedData object.
 
     Args:
-        aggregation_id: Give an explicit ID for the aggregation. If set to True, an
-            automatic ID based on existing realization uuid will be made.
+        aggregation_id: Give an explicit ID for the aggregation. If None, or not
+            provided, an automatic ID based on existing realization uuid will be made.
             Default is None which means it will be missing (null) in the metadata.
```

Comment:
Something is self-contradicting here. What will None actually give?

Reply:
Clarified